.changeset/early-baboons-fly.md

```diff
@@ -0,0 +1,6 @@
+---
+"wrangler": minor
+---
+
+feat: allow mounting Workers Assets on wildcard paths not just the root - e.g. `example.com/blog/*` can now serve assets.
+These assets will be served matching the assets dir, so `example.com/blog/*` will serve assets from the `blog` dir within your assets output folder.
```

Comment:
We have gone with the "mask" approach, not the "mount" approach, following the naming conventions used in previous discussions.

To be clear, if you have two routes:

- `example.com/blog/*` and
- `example.com/admin/*`

(and assuming the assets directory is `public`) then the first route would only ever serve assets from the `public/blog` directory and the second route would only ever serve assets from the `public/admin` directory, "for the same Worker deployment".

Reply:
yup, i forgot what the naming was and mount was what made sense in my head. naming is fun - i'll leave to korinne on whatever we want to describe it as publicly